.gitignore

```diff
@@ -10,7 +10,7 @@ target
 .idea
 *.iml
 *.log
-esvm
+/esvm
```

Comment:
It actually only matches esvm directories at the root of the repo. There was a bug previously that caused the elasticsearch plugin to build an esvm directory inside the src/ directory, but because of this ignore it was not noticed. It was however being watched by the ClusterManager :)